### PR TITLE
Some cleanup, mostly related to login

### PR DESCRIPTION
**Pull request by madelinecsmith**
* Cleaned up requirements.txt by removing unused requirements
* Fixed minor issue of logout button displaying when not logged in on the login page (fixes a failing test)

Based off of #35 -- sorry if I got the procedure for this slightly wrong.

### CODE REVIEW

File: src/nestdoorapp/forms.py [hunk 1]
```diff
@@ -1,8 +1,33 @@
...
+        fields = ["username", "email", "password1", "password2"]
+
+class UserAuthenticationForm(forms.ModelForm):
+    password = forms.CharField(label='Password', widget=forms.PasswordInput)
+
+    class Meta:
+        model = User
+        fields = ('username', 'password')
+
+    def clean(self):
```
Comment (lamontenunn) [inline src/nestdoorapp/forms.py:22]: form validator

**Review by lamontenunn — COMMENTED** [review]
lgtm

**Review by gtcolling — COMMENTED** [review]
good.

**Review by gtcolling — DISMISSED** [review]

**Review by SalvadorMedinaRuiz — APPROVED** [review]
All good

**Review by SalvadorMedinaRuiz — APPROVED** [review]
All good

**Review by gtcolling — APPROVED** [review]
Good.

**Review by SalvadorMedinaRuiz — APPROVED** [review]
All good

**Review by sjiang1 — APPROVED** [review]

**Pull request merged**